src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -195,10 +203,13 @@ void addContextProviders(ContextElementResponseVector& cerV, ContextRegistration
 
     if (cr.contextRegistrationAttributeVector.size() == 0)
     {
-      /* Registration without attributes */
-      for (unsigned int jx = 0; jx < cr.entityIdVector.size(); ++jx)
+      if (!limitReached)
       {
-        addContextProviderEntity(cerV, cr.entityIdVector.get(jx), cr.providingApplication);
+        /* Registration without attributes */
+        for (unsigned int jx = 0; jx < cr.entityIdVector.size(); ++jx)
+        {
+          addContextProviderEntity(cerV, cr.entityIdVector.get(jx), cr.providingApplication);
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/9101970658842d30fb0d1a0166d115e5d0358a8c